packages/runner/src/app/app.jsx

```diff
@@ -99,13 +159,28 @@ class App extends Component {
     $(win).on('resize', this._onWindowResize).trigger('resize')
   }
 
+  _onSpecListResizeStart = () => {
+    this.isSpecListResizing = true
+  }
+
   _onReporterResizeStart = () => {
     this.isReporterResizing = true
   }
 
+  _onSpecListResize = (specListWidth) => {
+    this.props.state.specListWidth = specListWidth
+    this.props.state.absoluteSpecListWidth = specListWidth
+
+    const $header = $(findDOMNode(this.refs.header))
+
+    this.props.state.updateWindowDimensions({
+      headerHeight: $header.outerHeight(),
+    })
+  }
+
   _onReporterResize = (reporterWidth) => {
-    this.props.state.reporterWidth = reporterWidth
-    this.props.state.absoluteReporterWidth = reporterWidth
+    this.props.state.reporterWidth = reporterWidth - this.props.state.specListWidth
+    this.props.state.absoluteReporterWidth = reporterWidth - this.props.state.specListWidth
```

Comment:
The trick is that if the feature flag is set to false, `state.specListWidth` is 0 and it's business as usual 😎